target/hexagon/idef-parser/parser-helpers.c

```diff
@@ -1877,6 +1890,7 @@ void gen_load(Context *c, YYLTYPE *locp, HexValue *num, HexValue *size,
 void gen_store(Context *c, YYLTYPE *locp, HexValue *num, HexValue *size,
                HexValue *ea, HexValue *src)
 {
+	(void)num;
```

Comment:
Please review if it makes sense to drop unused arguments.

Reply:
So `num` comes from the `fSTORE(NUM, ...)` and `fLOAD(NUM, ...)` macros but it's always set to `1`, is it for dealing with arrays? It might then be good to assert `num->imm.value == 1`.

Reply:
Mmh, let's just drop it from `gen_store` and `assert` that in the caller. If in the future we ever needs this we can fix it.

Reply:
Sounds good